packages/react/src/ErrorBoundaryGroup.spec.tsx

```diff
@@ -76,3 +76,11 @@ describe('useErrorBoundaryGroup', () => {
     )
   })
 })
+
+const TestWithErrorBoundaryGroup = withErrorBoundaryGroup(() => <>{TEXT}</>)
+describe('withErrorBoundaryGroup', () => {
+  it('should show children', () => {
+    const rendered = render(<TestWithErrorBoundaryGroup />)
+    expect(rendered.queryByText(TEXT)).toBeInTheDocument()
+  })
+})
```

Comment:
Can this represent the characteristics of ErrorBoundaryGroup?
I'm not sure if showing children is a big feature of ErrorBoundaryGroup!

Reply:
It's actually avoid codecov's failing, so I will improve in next Pull Request! Thanks!